test/e2e/windows/security_context.go

```diff
@@ -80,6 +102,7 @@ func runAsUserNamePod(username *string) *v1.Pod {
 			Name: podName,
 		},
 		Spec: v1.PodSpec{
+			NodeSelector: map[string]string{"kubernetes.io/os": "windows"},
```

Comment:
Just a note that it is a requirement to taint the nodes before running the windows tests. We didn't want to change the specs for all the tests to add the labels so this would be why it this wasn't there initially. I think because this tests is windows only and this pod has windows specific setting it ok to add here.

Reply:
We have mixed nodes(both linux+windows) in OpenShift cluster, so it became necessary to run e2es.

Reply:
make sense, for running tests on testgrid we taint the linux nodes (in this case it's only the control nodes): 

https://github.com/kubernetes-sigs/windows-testing/blob/ee418e4b7c5d6ce74cc90cf74e0b48754ce0463b/extensions/master_extension/v1/win-e2e-master-extension.sh#L52
https://github.com/kubernetes-sigs/windows-testing/blob/ee418e4b7c5d6ce74cc90cf74e0b48754ce0463b/gce/run-e2e.sh#L15

Adding the node selector here isn't a problem due to the fact this is in the `sig-windows` folder and only applicable to windows nodes.